### PR TITLE
added streak stats badge

### PR DESCRIPTION
**Pull request by hendrasob**
added streak stats badge for this repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -522,6 +522,10 @@ From [isitmaintained.com](http://isitmaintained.com):
 ```markdown
 ![1](https://github-readme-stats.vercel.app/api/top-langs/?username=Naereen&theme=blue-green)
 ```
+[![Anurag's github streak](https://github-readme-streak-stats.herokuapp.com/?user=Naereen&theme=blue-green)](https://github.com/DenverCoder1/github-readme-streak-stats)
```
Comment (Naereen) [inline README.md:525]: Please update `Anurag's` to `DenverCoder1's`, the author of this new badge app. Thanks!

**Review by Naereen — CHANGES_REQUESTED** [review]

**Comment by hendrasob** [comment]
Done friends

**Review by Naereen — APPROVED** [review]
Great change, thanks! I'll merge.

**Comment by Naereen** [comment]
Thanks again for the suggestion, and the modification, @hendrasob.

**Comment by hendrasob** [comment]
Okay friend, you're welcome.

**Comment by Naereen** [comment]
@hendrasob I just stumbled across your nice repo <https://github.com/hendrasob/badges>, if you want to add some of these nice badges to my (long) list, please feel free to submit a PR! I'll merge and be grateful :pray:! Regards from France, -- @Naereen

**Comment by hendrasob** [comment]
@Naereen 🇫🇷 this a good idea. I also very happy and grateful for your idea 👍. I will definitely submit a PR for your repo! Regards from Indonesia ~ @hendrasob 🇮🇩

**Comment by Naereen** [comment]
Thanks!

**Pull request merged**